pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/LLCSegmentCompletionHandlers.java

```diff
@@ -413,7 +413,7 @@ public String segmentCommitEndWithMetadata(@QueryParam(SegmentCompletionProtocol
         .segmentCommitEnd(requestParams, isSuccess, isSplitCommit,
             CommittingSegmentDescriptor.fromSegmentCompletionReqParamsAndMetadata(requestParams, segmentMetadata));
     final String responseStr = response.toJsonString();
-    LOGGER.info("Response to segmentCommitEnd:{}", responseStr);
+    LOGGER.info("Response to segmentCommitEnd for segment {} is:{}", segmentName, responseStr);
```

Comment:
"Response to segmentCommitEndWithMetadata for segment: '{}' is: {}"
(Make it different from segmentCommitEnd)